res/database/init.sql

```diff
@@ -1,3 +1,10 @@
+CREATE DATABASE  IF NOT EXISTS `bugcache` /*!40100 DEFAULT CHARACTER SET utf8mb4 */;
+USE `bugcache`;
+-- MySQL dump 10.13  Distrib 5.6.28, for debian-linux-gnu (x86_64)
+--
+-- Host: localhost    Database: bugcache
+-- ------------------------------------------------------
+-- Server version	5.7.11
```

Comment:
For the record, I used `mysqldump -d -h localhost -u root -pPASSWORD --skip-comments bugcache | grep -v "mysqldump: \\[Warning]" >db.sql`